bin/all_sky_search/pycbc_coinc_statmap_inj

```diff
@@ -118,25 +121,30 @@ if len(zdata) > 0:
     ifar = numpy.zeros(len(ftimes), dtype=numpy.float32)
     fap = numpy.zeros(len(ftimes), dtype=numpy.float32)
 
-    # We are relying on the injection data set to be the first one,
-    # this is determined
-    # by the argument order to pycbc_coinc_findtrigs
     pivot_ifo = zdata.attrs['pivot']
-    ifsort = ifdata.data['%s/time' % pivot_ifo].argsort()
-    ifsorted = ifdata.data['%s/time' % pivot_ifo][ifsort]
-    if_start, if_end = numpy.searchsorted(ifsorted, start), \
-                       numpy.searchsorted(ifsorted, end)
-
-    fisort = fidata.data['%s/time' % pivot_ifo].argsort()
-    fisorted = fidata.data['%s/time' % pivot_ifo][fisort]
-    fi_start, fi_end = numpy.searchsorted(fisorted, start), \
-                       numpy.searchsorted(fisorted, end)
+    if not args.ignore_inclusive:
+        # We are relying on the injection data set to be the first one,
+        # this is determined
+        # by the argument order to pycbc_coinc_findtrigs
+        ifsort = ifdata.data['%s/time' % pivot_ifo].argsort()
+        ifsorted = ifdata.data['%s/time' % pivot_ifo][ifsort]
+        if_start, if_end = numpy.searchsorted(ifsorted, start), \
+                           numpy.searchsorted(ifsorted, end)
+
+        fisort = fidata.data['%s/time' % pivot_ifo].argsort()
+        fisorted = fidata.data['%s/time' % pivot_ifo][fisort]
+        fi_start, fi_end = numpy.searchsorted(fisorted, start), \
+                           numpy.searchsorted(fisorted, end)
 
     # most of the triggers are here so presort to speed up later sorting
     bsort = back_stat.argsort()
     dec_fac = dec_fac[bsort]
     back_stat = back_stat[bsort]
 
+    if args.ignore_inclusive:
+        # If we are not calculating inclusive IFAR, we can skip this last bit
+        logging.info("Done!")
+        exit()
```

Comment:
I think using this is discouraged, so maybe just put the whole block in a `if not args.inclusive` or (better) move to a function and simply don't call it.

Line 181 and 182 also shouldn't be set if ignoring inclusive incase the presence of an [empty] array is taken to indicate that this is expected.

Reply:
The empty array is a good idea, and think that should fix the CI failure

Reply:
Is a for loop with an `if ...: continue` and an `else` okay? It removes the exit(), and prevents an (IMO) ugly big `if` block.

Reply:
Yes, I missed that this was in a for loop, `continue` is better then